pkg/pcr/measurement.go

```diff
@@ -363,29 +425,21 @@ func (s Measurements) Calculate(image []byte, initialValue uint8, hashFunc hash.
 	result[len(result)-1] = initialValue
 	logger.Printf("Set 0x -> 0x%X\n\n", result)
 
-	for _, measurement := range s {
-		if measurement == nil || measurement.IsFake() {
+	for _, m := range measureEvents {
+		data := m.CompileMeasurableData(image)
```

Comment:
Let's put this
if data == nil {
    continue
}

to 
if hashValue == nil {
    continue
}